extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -14,9 +14,10 @@
 Draft
 
 Open issues:
-Should Spherical Harmonics be supported or should clients be forced to evaluate the specular cubemap for diffuse contribution.  
-The question comes down to if resolution supplied by 9 coefficients SH is enough for most usecases?
-Since the SH are optional, and I believe the quality using 9 coefficients is reasonable,  I am leaning towards keeping them.   
+Should Spherical Harmonics be supported or should clients be forced to evaluate the specular cubemap for diffuse contribution?  
+The question comes down to if resolution supplied by 9 coefficients SH is enough for most usecases?  
+It could also be the case that a scene only wants diffuse contribution.  
+A link to an implementation that uses the cubemap for specular, diffuse and reflection would be beneficial.  
```

Comment:
By default three.js viewer experiences use a singular IBL cube map and generate the roughness mip chain and then uses that for both specular and diffuse environmental IBL lighting.

https://threejs.org/examples/webgl_materials_envmaps_exr.html

Click on the "debug" option to see how these are all stored in a single image.

Reply:
Notice the mip levels for Three.js stop getting smaller even at higher roughness levels:

![Screen Shot 2021-04-09 at 3 55 38 PM](https://user-images.githubusercontent.com/588541/114233959-
17281880-994c-11eb-8500-ae64cb4fa863.png)

This is why I do not want to link roughness levels to a cube map mip chain -- because clients can do better.

Reply:
The PMREM textures three generates are quite interesting. I wonder the what the tradeoffs of actually using a single image like this for this, created offline, for this extension instead of cubemaps with mip chain might be. It would certainly simplify some things, and make the input format requirements pretty flexible (we wouldn't really need to require KTX), but would certainly complicate both tooling and runtime requirements quite a bit since its being quite prescriptive...